src/medCore/database/medDatabaseController.cpp

```diff
@@ -587,13 +592,6 @@ bool medDatabaseController::moveDatabase( QString newLocation)
             qDebug() << "Restarting connection...";
             this->createConnection();
         }
-
-        // now delete the old archive
-        if(medStorage::removeDir(oldLocation))
```

Comment:
I would say you may still want to delete the old database if the folder you moved to was empty.